osc/core.py

```diff
@@ -3748,7 +3749,15 @@ def copy_pac(
         if force_meta_update or not found:
             print('Sending meta data...')
             u = makeurl(dst_apiurl, ['source', dst_project, dst_package, '_meta'])
-            http_PUT(u, data=src_meta)
+            http_PUT(u, data=meta)
+
+    if meta is None:
+        meta = show_package_meta(dst_apiurl, dst_project, dst_package)
+
+    root = ET.fromstring(meta)
+    if not root.find("scmsync") is None:
+        print("Note: package source is managed via SCM")
```

Comment:
so this breaks client side copy. 

at least with copypac client side copy should actually remove the scmsync tag, otherwise you can't commit afterwards..

Reply:
I can add that ...

Reply:
done